src/main/java/mate/academy/spring/dto/request/UserRequestDto.java

```diff
@@ -1,7 +1,16 @@
 package mate.academy.spring.dto.request;
 
+import com.sun.istack.NotNull;
+import javax.validation.constraints.Email;
+import javax.validation.constraints.Size;
+
 public class UserRequestDto {
+    @NotNull
+    @Email
+    @Size(min = 6)
```

Comment:
```suggestion
    @Email
```
you make all the ckecks in emain validator (if (email == null) and there is a pattern there)